src/main/java/s/volodymyr/onlinebookstore/dto/book/CreateBookRequestDto.java

```diff
@@ -22,6 +23,7 @@ public record CreateBookRequestDto(
         @PositiveOrZero
         BigDecimal price,
         String description,
-        String coverImage
+        String coverImage,
+        List<Long> categoryIds
```

Comment:
you may also validate this
```suggestion
        @NotEmpty List<@Positive Long> categoryIds
```